src/stores/Tabs/tab.js

```diff
@@ -296,6 +298,11 @@ export const Tab = types
       self.selected = ids;
     },
 
+    setSemanticSearch(searchText) {
```

Comment:
```suggestion
    setSearchText(searchText) {
```

Reply:
I would keep this open here, I think this search text usage is something best defined by the extension.